packages/components/src/components/hds/app-side-nav/index.ts

```diff
@@ -123,13 +123,11 @@ export default class HdsAppSideNav extends Component<HdsAppSideNavSignature> {
   }
 
   synchronizeInert(): void {
-    this._containersToHide?.forEach((element): void => {
-      if (this._isMinimized) {
-        element.setAttribute('inert', '');
-      } else {
-        element.removeAttribute('inert');
-      }
-    });
+    if (this._isMinimized) {
```

Comment:
[question/suggestion?] MDN [says](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/inert) that "The inert [...] attribute is a Boolean attribute". Does it mean this could be converted to `this._navWrapperBody?.setAttribute('inert', this._isMinimized);`?  (in case, test that it works, I didn't)

Reply:
Tested in chrome and it treated `inert="false"` as inert still, will leave it as is for now.

Reply:
Ohhh, intersting! 👍